kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/journal/journal.fxml

```diff
@@ -134,11 +136,12 @@
                         </HBox>
                      </top>
                      <center>
-                        <Pane fx:id="desktopSurfacePane" prefHeight="200.0" prefWidth="200.0" styleClass="desktop-area">
-                           <children>
-                              <Region fx:id="dropAnimationRegion" layoutX="32.0" layoutY="30.0" mouseTransparent="true" prefHeight="200.0" prefWidth="200.0" styleClass="search-drop-region" />
-                           </children>
-                        </Pane>
+                        <ScrollPane fx:id="desktopSurfaceScrollPane" focusTraversable="true" styleClass="desktop-scroll-pane">
+                           <AnchorPane fx:id="desktopSurfacePane" maxWidth="2800.0" maxHeight="1800.0" prefWidth="2800.0" prefHeight="1800.0" styleClass="desktop-area">
```

Comment:
According to the Figma design's the width would be 3 x 1080pixels or 3240.

Reply:
Thanks for the input Carl. The current workspace (Journal View) is actually set to 2800 x 1800 in the Figma designs (providing a screenshot below). Also the same values were stated in the Jira ticket in the requirements. Please let us know if the values are correct for and we will change them accordingly.

<img width="980" alt="figma-workspace-size" src="https://github.com/user-attachments/assets/54d0e727-e46a-4646-83f2-a293ebfe7386">